src/www/ui/template/include/foot.html.twig

```diff
@@ -4,6 +4,6 @@
    are permitted in any medium without royalty provided the copyright notice and this notice are preserved.
    This file is offered as-is, without any warranty.
 #}
-<script src="scripts/jquery-1.11.1.min.js" type="text/javascript"></script>
+<script src="scripts/jquery-3.2.0.min.js" type="text/javascript"></script>
```

Comment:
I opened everything, did many common actions I know and checked the browser console all the time. But I'm not yet familiar with all the functionality fossology has to offer. That's why more people should test this change as described above.